src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1273,6 +1273,23 @@ namespace winrt::TerminalApp::implementation
             exportTabMenuItem.Icon(exportTabSymbol);
         }
 
+        Controls::MenuFlyoutItem findMenuItem;
+        {
+            // "Split Tab"

```

Comment:
oop - this is find!

Reply:
Yeah sorry about that, obviously happened during copy-paste 😅